server/src/main/java/io/crate/auth/HttpAuthUpstreamHandler.java

```diff
@@ -149,6 +151,25 @@ private void handleHttpRequest(ChannelHandlerContext ctx, HttpRequest request) {
         }
     }
 
+    @Nullable
+    static Duration cacheControlFromRequest(HttpRequest request) {
+        String cacheControl = request.headers().get(HttpHeaderNames.CACHE_CONTROL.toString());
+        if (cacheControl != null) {
+            if (cacheControl.toLowerCase().startsWith("max-age:")) {
+                String value = cacheControl.substring(cacheControl.indexOf(":") + 1);
```

Comment:
```suggestion
            if (cacheControl.toLowerCase().startsWith("max-age=")) {
                String value = cacheControl.substring(cacheControl.indexOf("=") + 1);
```

fyi https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#directives

Reply:
👍